docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -65,7 +65,8 @@ notebooks:
     latestTagAsDefault: <Boolean>
 
     # 'enableAutoDois' is a notebook-wide setting that will affect ALL branches/tags; only set this as needed in a main branch
-    # to control whether DOIs are automatically issued for tags. Implicit default: True, can only be overridden as False
+    # to control whether DOIs are automatically issued for tags. Default: True.
+    # A value of true will enable automatic DOIs.
     # A value of false will disable automatic DOIs, DOIs will only be issued on demand.
```

Comment:
This line uses false with no capital F. The new line 68 has a capital T but the new line 69 doesn't. I prefer lower case for boolean unless there is a reason to prefer upper case.